scripts/generate_json_docs.py

```diff
@@ -296,9 +299,34 @@ def clean_source_path(module):
     return '%s.py' % (source_id.replace('.', '/'),)
 
 
+def get_examples_from_docstring(doc_str):
+    examples = doctest.DocTestParser().get_examples(doc_str)
+    example_str = ''
+    for example in examples:
+        example_str += '%s' % example.source
+        example_str += '%s' % example.want
+
+    return example_str.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
urlencode is different from htmlentities. I couldn't find an stdlib stuff for htmlentities :(.